### ISSUE TITLE
msg.text returns blank

### ISSUE THREAD

**Issue by Demo-Parcoursup** [ikvk/imap_tools#89]
Now I know this issue has been raised before on #86 and #80 but the advice was just "Well if the email has no text or html, it's normal that it doesn't return anything"

However here the email DOES have text, and using `msg.html` returns the message but not `msg.text`.

Using the linked eml and running the following code:
```
import imap_tools

with open('./example.eml', 'rb') as f:
    msg = imap_tools.MailMessage.from_bytes(f.read())
    print(msg.text)
    print(len(msg.text))
    print(msg.html)
```

I get this:
```
0
<div dir='auto'>Vous pouvez pas le voir mais en fichiers joints j'ai rajouter des photos compromettantes de Mathis Jung avec un incroyable accoutrement #supreme #TNF.<div dir="auto"><br></div><div dir="auto">Wow je n'aurai pas cru qu'il aurait autant de #Drip #Swag.</div><div dir="auto"><br></div><div dir="auto"><br></div></div>
```

So it manages to get the email's content as html code, but it returns nothing (length of 0) when trying to get it as plain text with `msg.text`. Any way I could fix that?

PS: Of course here I'm using the .eml to showcase the issue, but the same issue happens if I connect and fetch the email directly through `Mailbox`

[example.zip](https://github.com/ikvk/imap_tools/files/5820923/example.zip)

**Comment by Demo-Parcoursup** [comment]
I did some more debugging:

I tried sending the same email to a different gmail address (instead of sending it to myself), and testing if that .eml file worked. Same result, `msg.text` returns nothing.


However, if I download the .eml file that was **sent**, rather than the one that was **received**, this time the test above and `msg.text` work as intended. Though of course that doesn't fix the issue as I'd like to be able to read the text of received emails.

[example_sent.zip](https://github.com/ikvk/imap_tools/files/5821134/example_sent.zip)
